tasks/unit-tests/testdata/github_slack_map.yaml

```diff
@@ -37,7 +37,7 @@
 '@datadog/agent-cspm': '#k9-cspm-ops'
 '@datadog/telemetry-and-analytics': '#instrumentation-telemetry'
 '@datadog/asm-go': '#k9-asm-library-go'
-'@datadog/agent-build-and-releases': '#agent-build-and-releases'
-'@datadog/agent-ci-experience': '#agent-developer-experience'
-'@datadog/agent-developer-tools': '#agent-developer-experience'
+'@datadog/agent-delivery': '#agent-build-and-releases'
+'@datadog/agent-devx-infra': '#agent-developer-experience'
+'@datadog/agent-devx-loops': '#agent-developer-experience'
```

Comment:
```suggestion
'@datadog/agent-devx-infra': '#agent-devx-help'
'@datadog/agent-devx-loops': '#agent-devx-help'
```

Reply:
done in another PR https://github.com/DataDog/datadog-agent/pull/26965